src/components/ui/LogoHeader/LogoHeader.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import AnimatedNumbers from "react-animated-numbers";
+import * as S from "./LogoHeader.styles";
+
+export default function LogoHeader({ points = 0 }) {
```

Comment:
Change the name. It should be called something meaningful. What is this components purpose? To show a score from the quiz, right? So it can be called Score, or something like that